### ISSUE TITLE
Compile a new class with syntax error

### ISSUE THREAD

**Issue by JorneVL** [daishi4u/ForceCode#63]
When I try to save a new class I have created with ForceCode, it doesn't work and I don't receive a message.
I have added a small class where you don't receive feedback.
I use the latest version of VS Code and ForceCode.

```
public with sharing class TestUtilities
{
    public void test()
    {
        String a = String();
    }
}

```

If it is hard to give feedback why it doesn't save, maybe give a message that they need to create a empty class and try to save that.

**Comment by daishi4u** [comment]
This will be fixed in 3.8.0

**Comment by daishi4u** [comment]
This is now fixed, version 3.8.0 is uploaded. Please read the README as a LOT has changed including the project structure.
